packages/ng/forms/phone-number-input/phone-number-input.component.ts

```diff
@@ -67,6 +68,8 @@ export class PhoneNumberInputComponent implements ControlValueAccessor, Validato
 
 	@Input() autocomplete?: 'off' | 'tel';
 
+	autoPlaceholder = input<boolean>(true);
```

Comment:
I'm not a big fan of booleans with default value set to true, maybe we should use some kind of `disable...` input instead?

Reply:
correct, so it will be `noAutoPlaceholder`